sdks/python/src/opik/api_objects/dataset/converters.py

```diff
@@ -22,6 +22,16 @@ def to_pandas(
     return pd.DataFrame(new_item_dicts)
 
 
+def from_jsonl_file(file_path: str) -> List[Dict[str, Any]]:
```

Comment:
For consistency with other converting functions, `from_jsonl_file` should return `List[dataset_item.DatasetItem]`.
It's OK if it calls `from_json` under the hood and accepts `keys_mapping` and `ignore_keys` arguments.